config/detekt/detekt.yml

```diff
@@ -118,10 +118,10 @@ complexity:
     active: true
     threshold: 600
   LongMethod:
-    active: true
+    active: false
     threshold: 60
   LongParameterList:
-    active: true
+    active: false
```

Comment:
This probably isn't a good idea to simply disable the check 🤔 
How about we set `ignoreAnnotated: ['Composable']` instead if composables are the cause of detekt warning?